tools/nl/svindex_differ/run.sh

```diff
@@ -29,7 +31,7 @@ pip3 install -r nl_server/requirements.txt
 pip3 install -r tools/nl/svindex_differ/requirements.txt
 
 # Get the production embeddings.
-PROD=$(curl -s https://raw.githubusercontent.com/datacommonsorg/website/master/deploy/nl/embeddings.yaml | awk '{ print $2; }')
+PROD=$(curl -s https://raw.githubusercontent.com/datacommonsorg/website/master/deploy/nl/embeddings.yaml | awk '$1=="'$SIZE':"{ print $2; }')
```

Comment:
That's interesting, I would've thought `'` inside a `'` won't work :)

Reply:
Yup, that works and is needed to refer to the var `$SIZE`.